react-front-end/tsrc/search/SearchPageHelper.ts

```diff
@@ -378,3 +379,5 @@ export const getPartialSearchOptions = (
   options: SearchOptions,
   fields: SearchOptionsFields[]
 ) => pick(options, fields);
+
+export const WILDCARD_MODE_STORAGE_KEY = `wildcardMode_${AppConfig.baseUrl}`;
```

Comment:
I think it could be useful to flip these around such as:

```suggestion
export const WILDCARD_MODE_STORAGE_KEY = `${AppConfig.baseUrl}_wildcardMode`;
```

That way if have more, then we can easily delete for an institution. But, not to phased. :shrug: